packages/edit-post/src/components/secondary-sidebar/empty-outline-ilustration.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * WordPress dependencies
+ */
+import { Path, SVG, Line, Rect } from '@wordpress/components';
+
+export default function EmptyOutlineIlutration() {
```

Comment:
Is that a spelling error? Probably should fix and fix the references.

```suggestion
export default function EmptyOutlineIllustration() {
```

Reply:
Thank you, the typo was fixed 👍